doc/source/getting_started/faq.rst

```diff
@@ -12,6 +12,28 @@ Design Language (APDL), Ansys Fluent, and other Ansys products.
 You can use PyAnsys libraries within a Python environment of your choice
 in conjunction with external Python libraries.
 
+What Ansys license do I need to run the Geometry service?
+---------------------------------------------------------
+
+.. note::
+
+   Answered in https://github.com/ansys/pyansys-geometry/discussions/754
```

Comment:
```suggestion
   This question is answered in https://github.com/ansys/pyansys-geometry/discussions/754.
```
This suggestion assumes that is to be part of the doc. We want to use a complete sentence.